### ISSUE TITLE
Select - Axe Violation - ARIA attributes must conform to valid values

### ISSUE THREAD

**Issue by hegou33** [Sage/carbon#5289]
### Current behaviour

Select component has an axe violation when a value is selected, ensures all ARIA attributes have valid values.
![image](https://user-images.githubusercontent.com/38658070/177742570-acaa1e86-74cd-46af-8f20-ff5a34295771.png)

### Expected behaviour

No accessibility violations should occur.

### CodeSandbox or Storybook URL

https://carbon.sage.com/?path=/story/select--default-story

### Carbon Version

Latest

### What browsers are you seeing the problem on?

Chrome

### What Operating System are you seeing the problem on?

MacOS

### Confidentiality

- [X] I confirm there is no confidential or commercially sensitive information included.

**Comment by Parsium** [comment]
FE-5248
